nestkernel/nestmodule.cpp

```diff
@@ -1623,6 +1623,9 @@ NestModule::GetStructuralPlasticityStatus_DFunction::execute( SLIInterpreter* i
 /**
  * Enable Structural Plasticity within the simulation. This means, allowing
  * dynamic rewiring of the network based on mean electrical activity.
+ * Please note that in the current implementation of structural plasticity
+ * spikes could be delivered via connections that were not present at the 
+ * time of the spike in some cases.
```

Comment:
Perhaps it would be clearer if you wrote:

`Please note that, in the current implementation of structural plasticity, spikes could occasionally be delivered via connections that were not present at the time of the spike.`

What do you think?

Reply:
Great! thanks so much for checking this @sarakonradi